flixel/util/FlxTimer.hx

```diff
@@ -281,12 +309,35 @@ class FlxTimerManager extends FlxBasic
 	public function completeAll():Void
```

Comment:
Hm... Does `completeAll()` really need to be this complex? For tweens it's only 3 lines, this seems a little much.